curriculum/challenges/english/25-front-end-development/lab-palindrome-checker/657bdc55a322aae1eac3838f.md

```diff
@@ -0,0 +1,530 @@
+---
+id: 657bdc55a322aae1eac3838f
+title: Build a Palindrome Checker
+challengeType: 14
+dashedName: build-a-palindrome-checker
+---
+
+# --description--
+
+A <dfn>palindrome</dfn> is a word or phrase that can be read the same way forwards and backwards, ignoring punctuation, case, and spacing.
+
+**Note:** You'll need to remove **all non-alphanumeric characters** (punctuation, spaces and symbols) and turn everything into the same case (lower or upper case) in order to check for palindromes.
+
+**Objective:** Build an app that is functionally similar to <a href="https://palindrome-checker.freecodecamp.rocks" target="_blank" rel="noopener noreferrer nofollow">https://palindrome-checker.freecodecamp.rocks</a>
```

Comment:
we need to remove the link here because the preview will be available on click

```suggestion
**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
```